public_html/assets/css/style.css

```diff
@@ -0,0 +1,50 @@
+@import url(http://fonts.googleapis.com/css?family=Playfair+Display);
+#header {
+    background-color:#555555;
+    color:white;
+    text-align:center;
+    padding:5px;
+}
+#nav {
+    line-height:30px;
+    background-color:#FF99FF;
+    height:300px;
+    width:100px;
+    float:left;
+    padding:5px;	      
+}
+#section {
+    width:350px;
+    float:left;
+    padding:10px;	 	 
+}
+#footer {
+    background-color:#555555;
+    color:white;
+    clear:both;
+    text-align:center;
+   padding:5px;	 	 
+}
+#logo {
+	background: #555555;
+	position: aboslute;
+	margin: auto;
+	border-bottom-color: #000000;
+	border-bottom-style: solid;
+	border-top-color: #000000;
+	border-top-style: solid;
+}
+#title {
+	float:left;
+	margin:0 0 0 1.2em;
+	font-size:5em;
+	line-height:1.8em;
+	font-family:'Playfair Display';
+	font-weight:400;
+	letter-spacing:0.025em;
+}
+#title a {
+	color:#184668;
+	text-decoration:none;
+	transition:.5s;
+}
```

Comment:
typically we would use classes instead of id's

id's we use for specific targeting, especially for javascript